backend/pom.xml

```diff
@@ -158,6 +158,13 @@
             <version>${kessel.version}</version>
         </dependency>
 
+        <!-- Temporary dependency for the Errata subscriptions' migration service -->
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-csv</artifactId>
```

Comment:
We talked about using JSON in Slack, so I guess this won't be used.

Reply:
Removed!